front/components/authPage/LinkExpired.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <Layout>
+    <idea-card>
+      <div class="text-center" style="position: relative; width: 100%;">
+        <v-progress-linear
+          :indeterminate="true"
+          :active="sending"
+          :height="2"
+          absolute
+        ></v-progress-linear>
+
+        <h1 class="mt-5 font-weight-regular text-h1" style="font-size: 3rem;">
+          Looks like this link is expired.
+        </h1>
+        <p class="mb-10" style="font-size: 1.25rem; line-height: 1.2;">
+          The good news is that we just sent a new confirmation link to<br />
+          {{ email }}
+        </p>
+        <div class="ml-12 pl-2">
+          <img width="325" src="@/assets/images/bird-illustration.png" alt="" />
+        </div>
+        <div v-if="!hideSend" style="margin-top: -70px;">
+          <a href="#" @click.prevent="send2ndTime">
```

Comment:
why we need a tag here instead of just link-text  or vbtn text?

Reply:
For me it makes sense to use a tags for clickable elements, link-text is not a link, it just text styling, v-btn is an overkill for a simple anchor, plus it adds extra styles that are not needed.

Reply:
> link-text is not a link, it just text styling

So its supposed to be used for text link styling. Why a tag? Why not bind @click directly to link text?

Reply:
It's better practice to use appropriate tags for appropriate places (semantic HTML https://developer.mozilla.org/en-US/docs/Web/HTML/Element), anchors for clickable links, h1-6 for headers, ul li for lists, etc... At least for better readability, otherwise we can use just div for almost everything.

Reply:
The HTML <a> element (or anchor element), with its href attribute, creates a hyperlink to web pages, files, email addresses, locations in the same page, or anything else a URL can address.

Doesnt really fit into clickable element. Its not going to other places etc its just doing action, basically its button not anchor

Reply:
@aldarund Button has a style that represents button, when it's just a clickable text, it's an anchor.

Reply:
Hey @ezl we need an arbitrator here, we can't agree on using "a" tag here, and this is very serious issue...

Reply:
And link have style that represent link.  Using a for clickable elements is incorrect, a tags used for navigating as cited in the link that you provided.
https://blogs.ancestry.com/ancestry/2014/09/02/buttons-vs-anchors/

Reply:
@aldarund That's outdated and opinionated point of view, I don't agree with that. Everybody is using "a" tags for javascript actions. Using buttons for text links doesn't make sense it will require all the styling button has. It just simpler and make all the sense to mark clickable text with "a" tag than a button tag. 

Really why do you need to spend so much your and mine time on this absolutely irrelevant issue? What does it change for you, just to prove some point, for the project there is zero improvements if to use button instead of "a" tag, plus initially you suggested to use "span" for clickable element.

Reply:
The point is that there zero meaning of using here a tag. Its just there because its there? Its not fit semantically and don't bring any advantages here. Thats the point, that its not fit and don't bring anything

Reply:
Zero meaning? "a" tags means clickable link, it has 100% meaning to use it on clickable text. The only thing that doesn't match is that it has no link and a JS action instead, but button has move disadvantages with styling.

Reply:
But its not link, its action. So no. It could be just span with click and styling then, without 1 tag. Or properly styled button if we go proper way.  Its even bad from accessibility view. http://web-accessibility.carnegiemuseums.org/content/buttons/

Reply:
Link and action is basically the same, list as an action that navigates to other page. The look is more important that kind of action, button mostly look like button, when "a" is an anchor without any styling, and there is no styling just a clickable text. So "a" suits better for JS actions that button that require resetting all default styles it has and doesn't look like button.